pkg/config/reload.go

```diff
@@ -143,17 +146,23 @@ func (c *Config) ReloadPauseImage(newConfig *Config) error {
 }
 
 // ReloadPinnedImages updates the PinnedImages with the provided `newConfig`.
```

Comment:
optional: Might make sense to add a few words on what the bool this public method now returns indicates?

Reply:
Thx, I've rewritten this method and removed the `bool` return. Not log print happens in the same way as in other methods with usage of `logConfig()`